src/Components/Server/src/Circuits/CircuitRegistry.cs

```diff
@@ -105,9 +132,14 @@ public virtual Task DisconnectAsync(CircuitHost circuitHost, string connectionId
             return circuitHandlerTask;
         }
 
+        internal bool ContainsCircuit(string circuitId)
```

Comment:
Calling it `circuitId` here seems really confusing. The dictionary key is now the request token, not the circuit ID, isn't it?

Reply:
Yes, but in all effects is the string we use to identify the circuit